packages/create-remix/cli.ts

```diff
@@ -63,27 +69,48 @@ async function run() {
         ).dir
   );
 
+  let serverTemplate: string;
+  let getServerLangTemplate: () => string;
+  if (flags.adapter) {
+    let adapterPath = fse.existsSync(flags.adapter)
+      ? path.resolve(flags.adapter)
+      : resolveAdapterPackage(flags.adapter);
+    if (!fse.existsSync(path.join(adapterPath, "templates"))) {
+      console.warn(
+        `️🚨 WARNING: \`${flags.adapter}\` doesn't provide any templates, so this probably won't generate a working app 😬`
+      );
+    }
+    serverTemplate = path.join(adapterPath, "templates", "shared");
+    getServerLangTemplate = () =>
+      path.join(adapterPath, "templates", answers.lang);
+  } else {
+    let { server } = await inquirer.prompt<{ server: Server }>([
+      {
+        name: "server",
+        type: "list",
+        message:
+          "Where do you want to deploy? Choose Remix if you're unsure, it's easy to change deployment targets.",
+        loop: false,
+        choices: [
+          { name: "Remix App Server", value: "remix" },
+          { name: "Express Server", value: "express" },
+          { name: "Architect (AWS Lambda)", value: "arc" },
+          { name: "Fly.io", value: "fly" },
+          { name: "Netlify", value: "netlify" },
+          { name: "Vercel", value: "vercel" },
+          { name: "Cloudflare Workers", value: "cloudflare-workers" }
```

Comment:
Maybe add an Other option, if the user choose it ask for the adapter to use.

Reply:
I like that idea, seems more user-friendly than command-line flag

Reply:
I think it should have both, the flag is useful if you already know it exists, the option helps for discoverability

Reply:
Done